baseplate/lib/config.py

```diff
@@ -228,8 +228,8 @@ def File(mode: str = "r") -> Callable[[str], IO]:  # noqa: D401
     def open_file(text: str) -> IO:
         try:
             return open(text, mode=mode)
-        except IOError:
-            raise ValueError("could not open file: %s" % text)
+        except OSError:
```

Comment:
I didn't know the exception thrown by `open` changed. Nice catch

Reply:
looks like IOError and a bunch of others (socket.error! yay!) got moved under OSError. pyupgrade does this for us. v nice.